test/parking_location/configs/parking_location_choice.yaml

```diff
@@ -0,0 +1,26 @@
+SPECIFICATION: parking_location_choice.csv
+
+PREPROCESSOR:
+  SPEC: parking_location_choice_annotate_trips_preprocessor
+  DF: trips_merged
+  TABLES:
+    - land_use
+
+ALTS_PREPROCESSOR:
+  SPEC: parking_location_choice_annotate_alt_preprocessor.csv
+  DF: alt_tdd
+  TABLES:
+    - land_use
+
+
+# boolean column to filter choosers (True means keep)
+CHOOSER_FILTER_COLUMN_NAME: is_park_eligible
+CHOOSER_SEGMENT_COLUMN_NAME: parking_segment
+
+ALTERNATIVE_FILTER_COLUMN_NAME: is_parking_zone
+ALT_DEST_COL_NAME: parking_maz
+
+TRIP_DEPARTURE_PERIOD: 'stop_period'
+
+TRIP_ORIGIN: orig_maz
+TRIP_DESTINATION: dest_maz
```

Comment:
These should be origin and destination to be consistent with ActivitySim trip file.